packages/react/src/List/List.module.css

```diff
@@ -0,0 +1,35 @@
+.List {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  list-style: none;
+  gap: var(--base-size-8);
+  padding-inline-start: 0px;
+  font-family: var(--brand-body-fontFamily);
+  font-weight: var(--brand-text-weight-400);
+  font-size: var(--brand-text-size-200);
+  line-height: var(--brand-text-lineHeight-400);
```

Comment:
Quite a big difference between 400 and 200 in the type scale. This is rendering at 28px but the Figma spec is saying it should be 20px. Can we change this to whichever position maps to 20px, maybe 200 or 300?

Reply:
Updated in next push